Source/WebCore/platform/mediarecorder/MediaRecorderPrivateEncoder.cpp

```diff
@@ -463,7 +465,13 @@ void MediaRecorderPrivateEncoder::appendVideoFrame(const MediaTime& audioTime, R
         })->chainTo(WTFMove(producer));
     }
 
-    auto sampleTime = nextVideoFrameTime(audioTime);
+    auto sampleTime = time;
+    // We take the time before m_firstVideoFrameAudioTime is set so that the first frame will always apepar to have a timestamp of 0 but with a longer duration.
```

Comment:
```suggestion
    // We take the time before m_firstVideoFrameAudioTime is set so that the first frame will always appear to have a timestamp of 0 but with a longer duration.
```